main.js

```diff
@@ -78,12 +90,25 @@ function createWindow() {
 
 let createMenuTray = () => {
   tray = new Tray(__dirname + '/tray.png')
-  const contextMenu = Menu.buildFromTemplate([
+
+  const trayMenus = [
     { role: 'about' },
-    { label: 'Quit', click() { app.quit() } }
-  ])
+    { label: 'Quit', click() { app.quit() } },
+    { label: 'Set window always to top',
+      type: 'checkbox',
```

Comment:
No need for checkbox now.

Reply:
which means it is a one way option to set the window stay always on top. What if i want to revert it.